exercise-src/react-native/11-navigation/01-problem/src/App.tsx

```diff
@@ -0,0 +1,52 @@
+import type { FC } from "react"
+import { SafeAreaView } from "react-native"
+import { NavigationContainer } from "@react-navigation/native"
+import { createBottomTabNavigator } from "@react-navigation/bottom-tabs"
+import Icon from "react-native-vector-icons/Ionicons"
+
+import ThemeProvider, { useTheme } from "./design/theme/ThemeProvider"
+
+const AppTabs = createBottomTabNavigator()
+export const AppNavigator: FC = () => {
+  const theme = useTheme()
+
+  return (
+    <AppTabs.Navigator
+      initialRouteName="RestaurantsStack"
+      screenOptions={({ route }) => ({
+        headerStyle: {
+          backgroundColor: theme.palette.screen.main,
+        },
+        headerTitleStyle: {
+          color: theme.palette.screen.contrast,
+          ...theme.typography.title,
+        },
+        tabBarStyle: {
+          backgroundColor: theme.palette.screen.main,
+        },
+        tabBarActiveTintColor: theme.palette.primary.strong,
+        tabBarInactiveTintColor: theme.palette.screen.contrast,
+        tabBarIcon: ({ focused, color }) => {
+          //Exercise: Update the Icon component's name property based on the route
+          return <Icon name={icon} size={20} color={color} />
+        },
+      })}
+    >
+      {/*Exercise: Add the Screens for both the Restaurants and Setting tabs */}
+    </AppTabs.Navigator>
+  )
+}
```

Comment:
Chances are I'll remove the entire Navigator for the Exercise, if it's necessary to remove before writing the content for this section let me know.